JB_README.md

```diff
@@ -0,0 +1,39 @@
+# Extended update concept
+
+This concept is about policy files distribution approach where each client receives from hub only those files that are 
+specific to him. Atop of that, it uses autorun option to execute those. It is about extending existing update process to 
+copy client specific CFEngine policy files from separate repository directory on hub but not located in "masterfiles", since 
+those there will be copied to all clients.
+
+## Description
+
+Client uses it's IP address, hostname and tags file content to enlist subdirectories in hub repository (`$(sys.workdir)/repository/cfengine` 
+or check and adapt `jb_update_policy3.cf` file to your needs). Additional update policy file will look for `$(sys.workdir)/node_tags.dat`
+file where tags can be separated with white space character (\s, \n), coma (,) or semicolon (;). 
+
+Hub repository directory is added to to maintained CFEngine infrastructure, to be shared, with granted access and permissions adjusted (to 0600).
+
+## Schema
```

Comment:
No worries, I am glad you did.

That all sounds interesting.

Shoot me an email nick.anderson@cfengine.com and we can work out guest blog posts.